fpm/src/fpm.f90

```diff
@@ -1,34 +1,37 @@
 module fpm
 
-use fpm_strings, only: string_t, str_ends_with
-use fpm_backend, only: build_package
-use fpm_command_line, only: fpm_build_settings, fpm_new_settings, &
-                      fpm_run_settings, fpm_install_settings, fpm_test_settings
-use fpm_environment, only: run, get_os_type, OS_LINUX, OS_MACOS, OS_WINDOWS
-use fpm_filesystem, only: join_path, number_of_rows, list_files, exists, basename
-use fpm_model, only: srcfile_ptr, srcfile_t, fpm_model_t
-use fpm_sources, only: add_executable_sources, add_sources_from_dir, &
-                       resolve_module_dependencies
-use fpm_manifest, only : get_package_data, default_executable, &
-                         default_library, package_t
-use fpm_error, only : error_t
+use fpm_strings,      only : string_t, str_ends_with
+use fpm_backend,      only : build_package
+use fpm_command_line, only : fpm_build_settings, fpm_new_settings, &
+                             fpm_run_settings, fpm_install_settings, fpm_test_settings
+use fpm_environment,  only : run, get_os_type, OS_LINUX, OS_MACOS, OS_WINDOWS
+use fpm_filesystem,   only : join_path, number_of_rows, list_files, exists, basename, mkdir
+use fpm_model,        only : srcfile_ptr, srcfile_t, fpm_model_t
+use fpm_sources,      only : add_executable_sources, add_sources_from_dir, &
+                             resolve_module_dependencies
+use fpm_manifest,     only : get_package_data, default_executable, &
+                             default_library, package_t, default_test
+use fpm_error,        only : error_t
+use fpm_manifest_test, only : test_t
```

Comment:
I don't see any advantage in aligning those use statements, if we try to keep this alignment up we have to continuously modify this part and adding unnecessary changes to the git history.

Also, I don't think you use the `test_t` type here.

Reply:
Indeed, and in fact the `test_t` line is even aligned incorrectly in your alignment.

I strongly suggest not to unrelated stylistic changes in a PR. 

Essentially, try to keep each PR about one feature and concentrate on that. If you want to reformat some code, submit a separate PR. In this particular case, I suggest we do no reformat, but in some other cases it might make sense to improve formatting.